tests/appsec/iast/sink/test_xcontent_sniffing.py

```diff
@@ -0,0 +1,49 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+class Test_XContentSniffing:
+    """Test X Content options header missing detection."""
+
+    sink_fixture = SinkFixture(
+        vulnerability_type="XCONTENTTYPE_HEADER_MISSING",
+        http_method="GET",
+        insecure_endpoint="/iast/xcontent-missing-header/test_insecure",
+        secure_endpoint="/iast/xcontent-missing-header/test_secure",
+        data={"username": "shaquille_oatmeal", "password": "123456"},
+        location_map={"nodejs": "iast/index.js",},
```

Comment:
No location_map needed I think, unless this was actually tested in Node.

Reply:
fixed